resources/zib/zib-NursingIntervention.xml

```diff
@@ -0,0 +1,437 @@
+      <alias value="Aanvrager" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-HealthcareProvider-PractitionerRole" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-HealthcareProvider-Practitioner" />
+      </type>
+      <mapping>
+        <identity value="zib-nursingintervention-v3.2-2020EN" />
+        <map value="NL-CM:14.2.8" />
+        <comment value="Requester" />
+      </mapping>
+    </element>
+    <element id="CarePlan.contributor:requester.extension">
+      <path value="CarePlan.contributor.extension" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="CarePlan.contributor:requester.extension:contributorIsRequester">
+      <path value="CarePlan.contributor.extension" />
+      <sliceName value="contributorIsRequester" />
+      <min value="1" />
+      <max value="1" />
+      <type>
+        <code value="Extension" />
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/ext-NursingIntervention-ContributorIsRequester" />
+      </type>
+    </element>
+    <element id="CarePlan.addresses">
+      <path value="CarePlan.addresses" />
+      <short value="Indication" />
+      <definition value="The nursing-related problem (indication) providing cause for the intervention." />
+      <alias value="Indicatie" />
+      <max value="1" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Condition" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-Problem" />
+      </type>
+      <mapping>
+        <identity value="zib-nursingintervention-v3.2-2020EN" />
+        <map value="NL-CM:14.2.6" />
+        <comment value="Indication" />
+      </mapping>
+    </element>
+    <element id="CarePlan.goal">
+      <path value="CarePlan.goal" />
+      <short value="TreatmentObjective" />
+      <definition value="The description of the treatment goal that the intervention decision is based on." />
+      <alias value="Behandeldoel" />
+      <max value="1" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Goal" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-OutcomeOfCare" />

```

Comment:
Should actually refer to zib-TreatmentObjective according to zib?

Reply:
Oops

Reply:
Fixed